tests/certification/configuration/redis/redis_test.go

```diff
@@ -0,0 +1,354 @@
+func getUpdateEvent(key string, val string) configuration.UpdateEvent {
+	expectedUpdateEvent := configuration.UpdateEvent{
+		Items: map[string]*configuration.Item{
+			key: {
+				Value: val,
+			},
+		},
+	}
+	return expectedUpdateEvent
+}
+
+// Run redis commands and and add them in watcher
+func runRedisCommands(ctx flow.Context, updater *cu_redis.ConfigUpdater, messages *watcher.Watcher, basicTest bool) error {
+	scenarios := []struct {
+		cmd          []interface{}
+		want         [][]string
+		waitDuration time.Duration
+	}{
+		{
+			cmd:  []interface{}{"set", key1, "val1"},
+			want: [][]string{{key1, "val1"}},
+		},
+		{
+			cmd:  []interface{}{"mset", key1, "val1", key2, "val2"},
+			want: [][]string{{key1, "val1"}, {key2, "val2"}},
+		},
+		{
+			cmd:  []interface{}{"copy", key1, key2, "REPLACE"},
+			want: [][]string{{key2, "val1"}},
+		},
+		{
+			cmd:  []interface{}{"append", key1, "-append"},
+			want: [][]string{{key1, "val1-append"}},
+		},
+		{
+			cmd:  []interface{}{"setrange", key1, 4, "-offset"},
+			want: [][]string{{key1, "val1-offset"}},
+		},
+		{
+			cmd:          []interface{}{"expire", key1, 3},
+			want:         [][]string{{key1, "val1-offset"}, {key1, ""}},
+			waitDuration: 5 * time.Second,
+		},
+		{
+			cmd:  []interface{}{"set", key1, "val1"},
+			want: [][]string{{key1, "val1"}},
+		},
+		{
+			cmd:  []interface{}{"expire", key1, 10},
+			want: [][]string{{key1, "val1"}},
+		},
+		{
+			cmd:  []interface{}{"persist", key1},
+			want: [][]string{{key1, "val1"}},
+		},
+		{
+			cmd:  []interface{}{"expire", key1, -2},
+			want: [][]string{{key1, ""}},
+		},
+		{
+			cmd:  []interface{}{"set", key1, "val1"},
+			want: [][]string{{key1, "val1"}},
+		},
+		{
+			cmd:  []interface{}{"set", key2, "val2"},
+			want: [][]string{{key2, "val2"}},
+		},
+		{
+			cmd:  []interface{}{"rename", key2, key1},
+			want: [][]string{{key1, "val2"}},
+		},
+		{
+			cmd:  []interface{}{"move", key1, 1},
+			want: [][]string{{key1, ""}},
+		},
+		{
+			cmd:  []interface{}{"set", key1, 1},
+			want: [][]string{{key1, "1"}},
+		},
+		{
+			cmd:  []interface{}{"incr", key1},
+			want: [][]string{{key1, "2"}},
+		},
+		{
+			cmd:  []interface{}{"decr", key1},
+			want: [][]string{{key1, "1"}},
+		},
+		{
+			cmd:  []interface{}{"incrby", key1, 2},
+			want: [][]string{{key1, "3"}},
+		},
+		{
+			cmd:  []interface{}{"decrby", key1, 2},
+			want: [][]string{{key1, "1"}},
+		},
+		{
+			cmd:  []interface{}{"incrbyfloat", key1, 0.5},
+			want: [][]string{{key1, "1.5"}},
+		},
+		{
+			cmd:  []interface{}{"del", key1},
+			want: [][]string{{key1, ""}},
+		},
+	}
+
+	// Only run set and mset for basic test
+	if basicTest {
+		scenarios = []struct {
+			cmd          []interface{}
+			want         [][]string
+			waitDuration time.Duration
+		}{
+			{
+				cmd:  []interface{}{"set", key1, "val1"},
+				want: [][]string{{key1, "val1"}},
+			},
+			{
+				cmd:  []interface{}{"mset", key1, "val1", key2, "val2"},
```

Comment:
Currently, `Value` possible in configurationItem is `string` - how does setting via set or mset make a difference?

Reply:
Same events are generated in both set and mset. It's just with mset, you can set multiple keys at once